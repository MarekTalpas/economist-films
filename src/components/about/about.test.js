// @flow
import React from 'react';
import renderer from 'react-test-renderer';
import About from './about';

test('About renders correctly', () => {
  const tree : string = renderer.create(<About />).toJSON();
  expect(tree).toMatchSnapshot();
});
