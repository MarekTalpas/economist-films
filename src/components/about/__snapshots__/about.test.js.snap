exports[`test About renders correctly 1`] = `
<div>
  <span>
    This is the about page
  </span>
  <button
    type="button">
    <a
      onClick={[Function]}
      style={Object {}}>
      Back
    </a>
  </button>
</div>
`;
